tests/perf-test/BaseOptionsList.perf-test.tsx

```diff
@@ -67,7 +67,7 @@ describe('[BaseOptionsList]', () => {
     }
 
     test('Should render 1 section and a thousand items', () => {
-        measurePerformance(<BaseOptionsListWrapper />);
+        measureRenders(<BaseOptionsListWrapper />);
```

Comment:
Why was it changed in this PR? (relates to all these changes in this PR)

Reply:
Ah then I just wanted to fix to check and it seemed like a quick change but I can revert it